config/config.go

```diff
@@ -250,7 +250,7 @@ type (
 // New creates a config instance. It first loads the default configs. If the config path is not empty, it will read from
 // the file and override the default configs. By default, it will apply all validation functions. To bypass validation,
 // use DoNotValidate instead.
-func New(configPaths []string, _plugins []string, validates ...Validate) (Config, error) {
+func New(configPaths []string, _plugins []string, privKeyPath string, validates ...Validate) (Config, error) {
```

Comment:
we could add a `PrivKeyConfigFile` into `blockchain.Config`, then no need to change here

Reply:
done